src/Altinn.Apps/AppTemplates/AspNet/App.IntegrationTests/Data/apps/tdd/autodelete-true/config/texts/resource.nb.json

```diff
@@ -0,0 +1,193 @@
+{
```

Comment:
Being one of those who don't like too much testdata.. We can limit the number of entries in this file without affecting what we are able to test

Reply:
Removed all texts